content/post/une-version-plutôt-riche.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Une version plutôt riche"
+date: 2020-05-28T14:12:06+02:00
+publishdate: 2020-06-03
+draft: false 
+tags: ["rero-ils", "release", "recherche", "circulation", "métadonnées", "affichage"]
+slug: une-version-plutot-riche
+---
+
+Un nouvelle version de {{< smallcaps "rero ils" >}} a été publiée, plutôt riche
+en nouvelles fonctionnalités, ce qui nous fait bien entendu plaisir.
+Ci-dessous, un rapide résumé des nouveautés principales. Les notes de version
+complètes, mais techniques, en anglais, sont disponibles sur le projet GitHub :
+[`v0.9.0`][1].
+
+<!--more-->
+
+## Amélioration de la recherche
+
+Voilà longtemps que l'équipe des bibliothécaires insistaient sur ce point et
+les [tests des bibliothèques pilotes][2] l'on confirmé : pour la recherche dans
+un catalogue de bibliothèques, on s'attend à l'application par défaut de
+l'opérateur booléen `ET`, afin de réduire le bruit, quitte à n'avoir aucun
+résultat. C'est désormais chose faite. À l'occasion, l'ensemble de
+l'indexation des données et du fonctionnement de la recherche a été revisité
+et les améliorations sont sensibles.
+
+## Des silos fermés
+
+[UCLouvain][3], qui collabore activement au développement de 
+{{< smallcaps "rero ils" >}}, avait proposé l'implémentation des *silos 
+fermés*, à savoir la possibilité de bloquer les demandes ou de restreindre les
+lieux de retrait pour une localisation donnée. Après l'avoir proposé,
+[UCLouvain][3] l'a réalisé.
+
+{{< figure src="/img/rero-ils-paging.png" caption="Édition d'une localisation, avec des restrictions sur les lieux de retrait" >}}
```

Comment:
Cette capture d'écran mélange du texte en français et en anglais. Est-ce inévitable?

Reply:
Oui, tout n'est pas encore traduit.